portfolio/src/App.jsx

```diff
@@ -1,157 +1,145 @@
-              <a href="https://pwa-challenge1-b508091ee337.herokuapp.com/" target="_blank" class="project 4"></a>
-              <img src={Project4} alt="https://pwa-challenge1-b508091ee337.herokuapp.com/" />
-              <h3 className="text-2xl font-semibold mt-8">
-              This application features a number of data persistence techniques, insuring the application runs regardless of browser supported function. 
-              </h3>
-              <p className="text-gray-400 text-sm mt-4">https://github.com/tfkjosh/PWA-Challenge</p>
-              <div className="flex justify-evenly mt-12">
-                <button className="px-6 py-3 text-lg bg-gradient-to-t from-orange-500 rounded-full to-yellow-500 hover:from-orange-700 hover:to-yellow-700">
-                  Live preview
-                </button>
-                <button className="px-6 py-3 border rounded-full hover:border-orange-500 hover:text-yellow-500">
-                  Checkout github
-                </button>
-                </div>
-              </div>
+            <div className="flex flex-col sm:flex-col gap-12 mt-11">
+              {projects.map(
+                ({ imgSrc, description, githubHref, livePreviewHref }) => (
+                  <div className="border border-gray-500 rounded-md">
+                    <a href={githubHref} target="_blank">
+                      <img
+                        src={imgSrc}
+                        alt={githubHref}
+                        className="w-full h-auto"
+                      />
+                    </a>
+                    <div className="p-5">
+                      <h3 className="text-2xl font-semibold">{description}</h3>
+                      <div className="flex gap-4 justify-end mt-8">
+                        {livePreviewHref && (
+                          <a href={livePreviewHref} target="_blank">
+                            <button className="px-6 py-3 text-lg transition opacity-100 bg-gradient-to-t from-orange-500 rounded-full to-yellow-500 hover:opacity-90">
+                              Live preview
+                            </button>
+                          </a>
+                        )}
+
+                        <a href={githubHref} target="_blank">
+                          <button className="px-6 py-3 rounded-full opacity-100 border border-white">
+                            Checkout github
+                          </button>
+                        </a>
+                      </div>
+                    </div>
+                  </div>
+                )
+              )}
             </div>
           </div>
         </section>
+
         {/* Technologies */}
-        <section className="py-10" section id="Experience">
+        <section className="py-10" id="experience">
           <div className="container m-auto px-4">
             <h2 className="text-2xl font semi-bold">Experience</h2>
             <div className="mt-14">
```

Comment:
Another quick comment worth leaving - I would be very hesitant to ever add "Advanced" to your portfolio or resume until you have at least a few years under your belt working with that technology. For example, I would say just as of this year, after having worked with React for almost 3 years now, that I would consider myself to be an advance React programmer. 

Just a word of caution there. Suggesting that you are advanced at any of these technologies, especially just starting out your career, might signal to companies and to recruiters that you aren't aware of what you don't know. I'd say that you would probably want to mark yourself as a beginner at any of the technologies that you've been exposed to and then _maybe_ in the case that you feel _really_ comfortable with one of them, like React or Tailwind, that you could mark yourself as intermediate. But I would definitely save advanced or expert for many years down the line. Just thought I would spare you any misperception from recruiters and would definitely suggest making those changes before publishing this and adding it to your resume which you'll send to recruiters and companies